.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and test ContratosPR API
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
Should we call this `lint` instead of `build`?

Reply:
💯 agree.